Mapping/DocumentFinder.php

```diff
@@ -30,11 +30,18 @@ class DocumentFinder
      * Constructor.
      *
      * @param array $bundles Parameter kernel.bundles from service container.
+     * @param string $appRootClass A class that has to be in the app's root folder. By default, the app's kernel class is used.
```

Comment:
likes like the CS does not like something going over 120 characters.

Reply:
weird that it only is an issue for the 7.4 pipeline thought 😄